f2/src/components/phase-banner/index.js

```diff
@@ -5,22 +5,21 @@ import { Flex, Tag } from '@chakra-ui/core'
 import { Text } from '../text'
 import { Layout } from '../layout'
 
-export const PhaseBanner = props => (
+export const PhaseBanner = (props) => (
   <Flex bg="gray.100">
     <Layout>
       <Flex align="center" py={4}>
         <Tag
           fontFamily="body"
+          fontWeight="bold"
```

Comment:
do we need both family and weight?

Reply:
Yes! Family set the font family, as in "Noto Sans", we don't access the font family as "Noto Sans Bold" So weight is necessary!